Tools/autotest/default_params/copter-uavcan.parm

```diff
@@ -0,0 +1,97 @@
+CAN_P1_DRIVER   1.000000
```

Comment:
Look at how it is done on other vehicle type and config.
We should simply append only the config for uavcan not redefine all param ...